doc/sphinx/changes.rst

```diff
@@ -765,6 +765,51 @@ A few bug fixes and documentation improvements, in particular:
   fixes `#4741 <https://github.com/coq/coq/issues/4741>`_,
   by Helge Bahmann).
 
+Changes in 8.10.2
+~~~~~~~~~~~~~~~~~
+
+**Kernel**
+
+- Fixed a critical bug of template polymorphism and nonlinear universes
+  (`#11128 <https://github.com/coq/coq/pull/11128>`_,
+  fixes 8.6-regression `#11039 <https://github.com/coq/coq/issues/11039>`_,
```

Comment:
If I may, this one is not really a regression, and not from 8.6 anyway as we have found a way to instrument this bug in 8.5 too.